js/components/oppfolgingsdialog/utfylling/tiltak/liste/TiltakInformasjonKnapper.js

```diff
@@ -30,9 +30,11 @@ const TiltakInformasjonKnapper = (
     return (
         <TiltakButtonsRow>
             <div>
-                {!lagreSkjema && (aktoerHarOpprettetElement || !skalVurdereTiltak(element, fnr))
```

Comment:
Hvorfor splitte opp denne i to blokker? Er det forandring i / fiksing av logikken eller bare kodestil?

Reply:
Dette er en fiks av en bug. Når skjema vises så skal hverken Endre- eller "Gi din vurdering"-knapp vises. Med den logikken som var før ble alltid en av knappene vist.

Reply:
@kaja-nav Nice 😄